ireland.html

```diff
@@ -0,0 +1,56 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Nashville Software School</title>
+        <link rel="stylesheet" href="./styles/main.css">
+    </head>
+
+    <body>
+        <header class="header">
+            <!-- needs picture -->
+            <div><img class="logo" src="" alt=""></div>
+            <nav>
+                <div class="nav_button active"><a href="index.html">Japan</a></div>
+                <div class="nav_button"><a href="norway.html">Norway</a></div>
+                <div class="nav_button"><a href="ireland.html">Ireland</a></div>
+                <div class="nav_button"><a href="india.html">India</a></div>
+            </nav>
+            <div><img class="country_img" src="./images/ireland.jpg" alt=""></div>
+            <h1>Ireland</h1>
+        </header>
+
+
+        <main id="container">
+            <article class="lists">
+                <section class="lists__cities">
+                    <ul>
+                        <li>Dublin</li>
+                        <li>Belfast</li>
+                        <li>Cork</li>
+                    </ul>
+                </section>
+                <section class="lists__landmarks">
+                    <ul>
+                        <li>Rock of Cashel</li>
+                        <li>Newgrange Tomb</li>
+                        <li>Blarney Stone and Castle</li>
+                    </ul>
```

Comment:
You'll want to add an h2 header for your list here.
Example: Famous Landmarks

Reply:
updated previous comment to show recommended code.